koku/masu/external/ros_report_shipper.py

```diff
@@ -142,8 +142,7 @@ def send_kafka_message(self, msg):
 
     def build_ros_msg(self, presigned_urls, upload_keys):
         """Gathers the relevant information for the kafka message and returns the message to be delivered."""
-        with ProviderDBAccessor(self.provider_uuid) as provider_accessor:
-            cluster_alias = provider_accessor.get_provider_name()
+        cluster_alias = get_cluster_id_from_provider(self.provider_uuid)
```

Comment:
Is this still getting the cluster alias here? Doesn't this return the cluster uuid instead?

Reply:
good point this might need to be 
```suggestion
        cluster_id = get_cluster_id_from_provider(openshift_provider_uuid)
        cluster_alias = get_cluster_alias_from_cluster_id(cluster_id)
```